apps/src/flowlab/views/flow/nodes/AiNode.tsx

```diff
@@ -48,7 +108,7 @@ function AiNode({
         data.askedText;
       console.log('Ask chat:', text);
 
-      const response = await askAi(text);
+      const response = await askAi(text, channelId, uploadedFiles.current);
```

Comment:
Should `uploadedFiles` be cleared after each AI request? Or do we want to keep them "stored" in the node?

Reply:
I did try clearing them after each request, but then the question would refer to something that was no longer attached and re-executing the query stopped giving a good answer, so decided to leave them stored in the node.

Longer term it probably makes sense to have each file in its own node which is then attached to the AI node.